cli/flox/doc/flox-config.md

```diff
@@ -7,69 +7,112 @@ header: "flox User Manuals"
+4. Environment variables.
+   All config options may be set by prefixing with `FLOX_` and using
+   SCREAMING_SNAKE_CASE.
+   For example, `disable_metrics` may be set with `FLOX_DISABLE_METRICS=true`.
 
-Without any flags or `-l` or `--list` shows all configurable options
-with their computed value.
+The last occurence is used as the final value.
+
+`flox config` commands that mutate configuration always write to
+`${FLOX_CONFIG_HOME:-$XDG_CONFIG_HOME}/flox/flox.toml`.
 
 ## Key Format
 
-`<key>` supports dot-separated queries for nested vaules, e.g.
+`<key>` supports dot-separated queries for nested values, for example:
 
 ```
-flox config --set 'nix.access_tokens."github.com"' "ghp_xxx"`
+flox config --set 'trusted_environments."owner/name"' trust
 ```
 
-## Config Format
+# OPTIONS
 
-All config keys can be listed with
+## Config Options
 
-```
-flox config
-```
+`-l`, `--list`
+:   List the current values of all options.
 
-The dispalyed values are resolved by reading:
+`-r`, `--reset`
+:   Reset all options to their default values without further confirmation.
 
-1. package defaults from `$PREFIX/etc/flox.toml`
-2. installation defaults from `/etc/flox.toml`
-3. user customizations from `$HOME/.config/flox/flox.toml`
-4. environment variables
+`--set <key> <value>`
+:  Set `<key> = <value>` for string values
 
-where the last occurence is used as the final value.
+`--set-number <key> <value>`
+:  Set `<key> = <value>` for number values
 
-`flox config` commands that mutate configurations always write to `$HOME/.config/flox/flox.toml`.
+`--set-bool <key> <value>`
+:  Set `<key> = <value>` for boolean values
 
-# OPTIONS
+`--delete <key>`
+:   Delete config key
 
 ```{.include}
 ./include/general-options.md
-./include/development-options.md
 ```
 
-## Config Options
+# SUPPORTED CONFIGURATION OPTIONS
+
+`config_dir`
+:   Directory where flox should load its configuration file (default:
+    `$XDG_CONFIG_HOME/flox`).
+    This option will only take effect if set with `$FLOX_CONFIG_HOME`.
+    `$FLOX_CONFIG_DIR` and `config_dir` are ignored.
+
+`cache_dir`
+:   Directory where flox should store ephemeral data (default:
+    `$XDG_CACHE_HOME/flox`).
+
+`data_dir`
+:   Directory where flox should store persistent data (default:
+    `$XDG_DATA_HOME/flox`).
+
+`disable_metrics`
+:   Disable collecting and sending usage metrics.
+
+`floxhub_token`
+:   Token to authenticate on FloxHub.
 
-[ (\--list|-l) ]
-:   List the current values of all configurable parameters.
+`search_limit`
+:   How many items `flox search` should show by default.
 
-[ (\--reset|-r) ]
-：  Reset all configurable parameters to their default values without further confirmation.
+`trusted_environments`
+:   Remote environments that are trusted for activation.
+    Contains keys of the form `<owner/name>` that map to either `trust` or
+    `deny`.
 
-[ \--set `<key>` `<value>` ]
-：  Set `<key> = <value>` for string values
+## Options used internally
```

Comment:
They show up in the output of `flox config`, so I think they should be explained in the man page. But maybe it would be better for them to be in neither

Reply:
I guess we can wrap them with options so they dont show up unless there's values in them

Reply:
I think internal options shouldn't be user visible, so let's remove them in both places.

Reply:
Fixing